### PR TITLE
Create Story promo component (ws frontpage card)

### LINKED ISSUE #391: Create homepage list card component @600px
Current UX for v1 of the homepage includes a flat list of articles grouped by category. Each article will be represented on the list by a card containing:
* image
* headline
* synopsis
* timestamp

The task in this issue is to set up the basic file structure and config for a card component in Psammead, including a Storybook story with a basic html output of the elements above. Pairing with UX and iterative development of this component is encouraged 😉 

TBD by UX:
* Basic card UI
* Hover states
* Clickable areas
* Screen reader text

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by dr3**
Resolves #391

**Overall change:** Creates initial component at 600px design

**Code changes:**

- Creates initial component at 600px design

image

**What doesn't this PR do?**
- Make it a link
- Make it hover/focusable
- Localise fonts
- Handle the design for the "Top" top story
- Add a grid fallback

**Testing Notes:**
Only Storybook check required

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-story-promo/README.md
Comment (pjlee11) [inline packages/components/psammead-story-promo/README.md:1]: Badges looking a little strange when on "view file" display

File: packages/components/psammead-story-promo/src/index.jsx
Comment (pjlee11) [inline packages/components/psammead-story-promo/src/index.jsx]: Why is this changing from `Pica` to `GreatPrimer` between breakpoints, this is not the use case for the typography group specs and should not be implemented this way. If this was a UX request we need to push back

File: packages/components/psammead-story-promo/src/index.stories.jsx
Comment (pjlee11) [inline packages/components/psammead-story-promo/src/index.stories.jsx]: Based on the [docs](https://github.com/bbc/psammead/blob/latest/packages/components/psammead-image/README.md) this is meant to be the `px` value of the image not the css being applied

File: packages/components/psammead-story-promo/README.md
Comment (dr3) [reply packages/components/psammead-story-promo/README.md:1]: Yeah, some dont work on an unpublished package, should be fine once published

File: packages/components/psammead-story-promo/src/index.stories.jsx
Comment (dr3) [reply packages/components/psammead-story-promo/src/index.stories.jsx]: Fixed :)

File: packages/components/psammead-story-promo/src/index.jsx
Comment (dr3) [reply packages/components/psammead-story-promo/src/index.jsx]: UX will get back to us later with these corrected :)

**Comment by dr3** [comment]
Blocked on UX typography

File: packages/components/psammead-story-promo/src/index.stories.jsx
Comment (pjlee11) [inline packages/components/psammead-story-promo/src/index.stories.jsx]: Not convinced on the naming `TextComponent`

Comment (pjlee11) [reply packages/components/psammead-story-promo/src/index.stories.jsx]: Text is containing a headline, summary and timestamp. Could this be `PromoInfo` or similar

File: packages/components/psammead-story-promo/src/index.stories.jsx
Comment (pjlee11) [inline packages/components/psammead-story-promo/src/index.stories.jsx]:
```suggestion
      return <StoryPromo image={ImageComponent} info={info} />;
```

Same reasoning as above, `Text` doesn't feel like the right naming

Comment (dr3) [reply packages/components/psammead-story-promo/src/index.stories.jsx]: renamed :)

File: packages/components/psammead-story-promo/src/index.stories.jsx
Comment (dr3) [reply packages/components/psammead-story-promo/src/index.stories.jsx]: renamed :)

Comment (DenisHdz) [reply packages/components/psammead-story-promo/src/index.stories.jsx]: Do we need to pass the image `width`? Are we really using that value?

Comment (dr3) [reply packages/components/psammead-story-promo/src/index.stories.jsx]: @DenisBBC Annoyingly its a required prop https://github.com/bbc/psammead/blob/latest/packages/components/psammead-image/README.md at the moment.

**Comment by DenisHdz** [comment]
LGTM once the conflict with `Pica` to `GreatPrimer` is solved 👍

**Comment by dr3** [comment]
@DenisBBC @pjlee11 Chatted with UX and weve decided to use Pica at all breakpoints for now :)

File: packages/components/psammead-story-promo/src/index.jsx
Comment (dr3) [reply packages/components/psammead-story-promo/src/index.jsx]: Resolved in https://github.com/bbc/psammead/pull/453#issuecomment-486170986

File: packages/components/psammead-story-promo/src/index.stories.jsx
Comment (pjlee11) [inline packages/components/psammead-story-promo/src/index.stories.jsx]: Thoughts on this being `Image` so that when it's used lower down it is `<Image>` and not `` which could be confused with the real HTML ``?

Comment (dr3) [reply packages/components/psammead-story-promo/src/index.stories.jsx]: SGTM

Comment (DenisHdz) [reply packages/components/psammead-story-promo/src/index.stories.jsx]: It makes sense 👍

Comment (dr3) [reply packages/components/psammead-story-promo/src/index.stories.jsx]: Done :)

**Review by pjlee11 — DISMISSED** [review]
LGTM 👍

**Review by DenisHdz — DISMISSED** [review]
Awesome

**Comment by jamesbrumpton** [comment]
There are currently issues on Afaan Oromoo, Igbo, Indonesian & Ukrainian. For example - `Прем&#39;єр спочатку нервував, проте швидко опанував ситуацію.` It shouldn’t have `&#39;` in there.
StoryPromo_-_default_⋅_Storybook

**Review by pjlee11 — APPROVED** [review]

**Review by DenisHdz — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM. Happy for this to be merged.

**Pull request merged**
